config.py

```diff
@@ -1427,9 +1453,72 @@ def get_sessions():
         return sessions_
 
 
-def get_runs() -> list:
+def get_runs_all_subjects() -> dict:
+    '''Returns a dict of runs present in the bids_path
+    for each subject asked in the configuration file
+    (and not for each subject present in the bids_path).'''
+    # We cannot use get_subjects() because if there is just one subject
+    # we need to creat the full list of ignore_subjects
```

Comment:
```suggestion
    # we need to create the full list of ignore_subjects
```

but move this comment below as here it's not yet understandable what you mean